src/allotropy/parsers/appbio_quantstudio_designandanalysis/appbio_quantstudio_designandanalysis_contents.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+
+from allotropy.parsers.utils.values import (
+    assert_not_empty_df,
+    assert_not_none,
+)
+from allotropy.types import IOType
+
+
+class DesignQuantstudioContents:
+    def __init__(self, contents: IOType) -> None:
+        self.raw_contents = pd.read_excel(contents, header=None, sheet_name=None)
```

Comment:
contents and raw_contents are not accessed after they are used to create header and data, they should not be stored as member variables. Instead they should be normal variables, and should be passed into _get_header and _get_data